apps/ledger-live-desktop/src/renderer/components/OperationsList/DateCell.jsx

```diff
@@ -43,7 +59,19 @@ class DateCell extends PureComponent<Props> {
           {text ||
             t(operation.hasFailed ? "operationDetails.failed" : `operation.type.${operation.type}`)}
         </Box>
-        <OperationDate date={operation.date} />
+        {editable ? (
+          <Box fontSize={3} color="palette.text.shade80">
+            <Box ff="Inter|SemiBold" fontSize={3} color="palette.text.shade80" style={ellipsis}>
+              {// display pending icon (less than 5 minutes: loading spinner; more than 5 minutes: yellow warning icon)
+              this.pendingLoadingIcon(new Date() - operation.date > 1000 * 5 * 60)}
+              <div style={{ verticalAlign: "middle", display: "inline-block", fontSize: "10px" }}>
```

Comment:
We should try to avoid using div, better using Box with the right props

Reply:
fixed